### ISSUE TITLE
Question about the motivation of the loss function when finetuning.

### ISSUE THREAD

**Issue by Sutongtong233** [williamyang1991/DualStyleGAN#41]
Hi, I find the loss function in Eq.(4) containing many parts. In styleGAN, only L_adv is used. I wonder if there is any motivation of using such a combination of loss. Is there any ablation study for IDloss? I only find ablation for reg_loss..

**Comment by williamyang1991** [comment]
StyleGAN's task is face synthesis. It goal is just to ensure the realism of the output with adv loss.
Our task is style transfer. Our goal is to achieve the content consistency with the content loss as well as style consistency with the tyle loss. The adv loss is to make the results more vivid. ID loss is commonly used in face editing to preserve the original face features, therefore, we use it as well. 

I tried different settings of ID loss weight for each style but I only saved the best model I finally found (therefore, there is no ablation study for ID loss). I used a higher ID loss weight for the caricature style, which I found could better preserve the identity. Maybe you can train with different ID loss weights to see its effects.

**Comment by Sutongtong233** [comment]
Thank You For Replying.
